Tools/Scripts/webkitpy/api_tests/manager.py

```diff
@@ -150,8 +151,10 @@ def _print_tests_result_with_status(self, status, runner):
                 self._stream.writeln('')
 
     def _initialize_devices(self):
-        if 'simulator' in self._port.port_name:
-            SimulatedDeviceManager.initialize_devices(DeviceRequest(self._port.DEVICE_TYPE, allow_incomplete_match=True), self.host, simulator_ui=False)
```

Comment:
Maybe we should keep this around as an `elif`, otherwise we're breaking watchOS. I know at least some API tests worked on watchOS at some point, although infrastructure doesn't run it.